src/appleseed-max-impl/appleseeddisneymtl/appleseeddisneymtl.h

```diff
@@ -174,6 +174,16 @@ class AppleseedDisneyMtl
         const char*         name);
 };
 
+//
+// Texture parameter accessor class declaration
+//
+
+class TexturePBAccessor
+    : public PBAccessor
+{
+public:
+    void Set(PB2Value& v, ReferenceMaker* owner, ParamID id, int tabIndex, TimeValue t) override;
+};
 
```

Comment:
I could probably put this class into utilities for other materials to access it.

Reply:
Yes, absolutely! You can add it to `utilities.{h,cpp}` in the "Parameter blocks functions" section. However it looks like the implementation of this class is specific to the Disney material right now.

Reply:
I'll put update_buttons function into utilities.